comp/core/gui/guiimpl/views/private/css/stylesheet.css

```diff
@@ -202,6 +202,23 @@ a {
   -webkit-box-shadow: 0px 0px 15px rgba(0, 0, 0, 0.22);
   display: none;
 }
+
+#main .error_page {
+  width: auto;
+  height: auto;
+
+  /* top: 20px; */
+  /* left: 20px;
+  border: 1px solid #e4e4e4; */
+  background-color: #f9ecec;
+  /* box-shadow: 0px 0px 15px rgba(0, 0, 0, 0.22);
+  -moz-box-shadow: 0px 0px 15px rgba(0, 0, 0, 0.22);
+  -webkit-box-shadow: 0px 0px 15px rgba(0, 0, 0, 0.22); */
+  /* text-align: center; */
+  /* margin: 5vh; */
```

Comment:
Do we need all of the commented out styling?

Reply:
Not at all, cleaned up 🧹